geekhours.md

```diff
@@ -0,0 +1,97 @@
+# Geek Hours Command Examples
+
+## Help messages
+```
+$ geekhours -h
+usage: geekhours [-h] {course,done} ...
+
+Geekhours is a management tool for the study time.
+
+optional arguments:
+  -h, --help     show this help message and exit
```

Comment:
remove "and exit" because of self-evident.

Reply:
Fixed.